documentation/src/main/asciidoc/topics/proc_using_kyro_marshaller.adoc

```diff
@@ -1,15 +1,12 @@
-[id='kryo_marshalling']
-= Using the Kryo Marshaller
-[NOTE]
-====
-Kryo marshaller is now deprecated and planned for removal. You should replace any Kyro marshaller usage with ProtoStream marshalling.
-====
+[id='using-kryo-marshaller_{context}']
+= Using the Kryo marshaller
+{brandname} provides a marshalling implementation that uses Kryo libraries, which is now deprecated.
 
-{brandname} provides a marshalling implementation that uses Kryo libraries.
+.Prerequisites
 
-.Prerequisites for {brandname} Servers
+* Remote caches
 
-To use Kryo marshalling with {brandname} servers, add a JAR that includes the
+To use Kryo marshalling with {brandname} Server, add a JAR that includes the
```

Comment:
the link to infinispan-marshaller-kryo-bundle below is broken. It has the wrong version `13.0`. it should be `13.0.0-SNAPSHOT` (or 13.0.0.Final)

Reply:
thanks, Pedro. I think we'll just remove the Kyro and Protostuf marshaller docs altogether.